frontend/src/pages/landingPageTemplate/LandingPageTemplate.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from "react";
+
+import { LandingPageElement } from '../../components/landing/Landing';
+
+import Layout from "../../components/layout/Layout";
+import Spinner from "../../components/atoms/spinner/Spinner";
+
+import styles from "./LandingPageTemplate.module.css";
+import { runningOnWebRenderer } from "../../helpers/utils";
+import closePtOsLandingWindow from "../../services/closePtOsLandingWindow";
+
+export type Props = LandingPageElement;
```

Comment:
Since this is the only place that uses LandingPageElement we can inline the props here, I also changed the message type to ReactNode (see my other comments)

```suggestion
export type Props = {
  title: string;
  message: React.ReactNode;
  prompt: ReactElement;
  image: string;
  urlInfo: UrlData;
  buttonLabel?: string;
};
```